Source/OxyPlot/Series/AreaSeries.cs

```diff
@@ -324,5 +349,22 @@ protected internal override void UpdateMaxMin()
             base.UpdateMaxMin();
             this.InternalUpdateMaxMin(this.ActualPoints2);
         }
+
+        /// <summary>
+        /// Gets the points when <see cref="P:ConstantY2" /> is used.
+        /// </summary>
+        /// <returns>A sequence of <see cref="T:DataPoint"/>.</returns>
+        private IEnumerable<DataPoint> GetConstantPoints2()
+        {
+            var actualPoints = this.ActualPoints;
```

Comment:
base the constant second data set on the first and last point of the first data set